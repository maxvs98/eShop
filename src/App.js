import React, { Component } from 'react';
import { connect } from 'react-redux';
import { Container } from 'semantic-ui-react';
import { setProducts } from './actions/products';
import axios from 'axios';
import Menu from './components/Menu';
import ProductCard from './components/ProductCard';
import { Card } from 'semantic-ui-react';

class App extends Component {
  componentWillMount() {
    const { setProducts } = this.props;
    axios.get('/products.json').then(({ data }) => {
      setProducts(data);
    });
  }

  render() {
    const { products, isReady } = this.props;
    return (
      <Container>
        <Menu />
        <Card.Group itemsPerRow={4}>
          {!isReady
            ? 'загрузка'
            : products.map((product, i) => (
              <ProductCard key={i} {...product} />
            ))}
        </Card.Group>
      </Container>

    );
  }

}

const mapStateToProps = ({ products }) => ({
  products: products.items,
  isReady: products.isReady
});

const mapDispatchToProps = dispatch => ({
  setProducts: products => dispatch(setProducts(products))
});

export default connect(mapStateToProps, mapDispatchToProps)(App);
