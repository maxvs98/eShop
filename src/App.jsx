import React, { Component } from 'react';
import Shop from './components/Shop';
import Users from './components/Users';
import Home from './components/Home';
import LoginComp from './components/LoginComp';
import PrivateRoute from './containers/PrivateRoute';

import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link,
  Redirect,
  withRouter
} from "react-router-dom";

const fakeAuth = {
  isAuth: false,
  authenticate(callback) {
    this.isAuth = true;
    setTimeout(callback, 100);
  },
  logout(callback) {
    this.isAuth = false;
    setTimeout(callback, 100);
  }
};

class App extends Component {
  render() {
    return (
      <Router>
        <Switch>
          <Route path="/" exact component={Home}/>
          <Route path="/shop" component={Shop}/>
          <Route path="/login" component={LoginComp} />
          <PrivateRoute path="/users" component={Users} />
        </Switch>
      </Router>
    );
  }
}

export default App;
