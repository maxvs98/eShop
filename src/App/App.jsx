import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route,
} from 'react-router-dom';
import Shop from '../Components/Shop/ShopComponent';
import Users from '../Components/Users/UsersComponent';
import Home from '../Components/Home/HomeComponent';
import Login from '../Components/Login/LoginContainer';
import Registration from '../Components/Registration/RegistrationContainer';
import PrivateRoute from '../Components/PrivateRoute/PrivateRouteContainer';

const App = () => (
  <Router>
    <Switch>
      <Route path="/" exact component={Home} />
      <Route path="/shop" component={Shop} />
      <Route path="/login" component={Login} />
      <Route path="/registration" component={Registration} />
      <PrivateRoute path="/users" component={Users} />
    </Switch>
  </Router>
);

export default App;
