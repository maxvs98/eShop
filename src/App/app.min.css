body{font:100% "Montserrat", sans-serif;color:#333;background-color:#e4e5e7}th{text-align:center}td{text-align:center}.header__content{height:105px;background-color:white;padding:30px 0px}.filter__content{height:50px;background-color:white;margin-bottom:50px}.menu{margin-bottom:20px;background-color:white}a:active,a:hover,a{text-decoration:none;color:#000}.logo{width:177px;height:37px}.cards__content{margin-bottom:50px}.menuIcon{margin-top:10px;width:22px;height:16px;background:url("/MenuIcon.png") no-repeat center}.menu{padding-top:10px}.menu__item{float:left;margin-left:42px;font-size:14px;font-weight:700}.menu__item:hover{color:#b9bed1}.cart__icon{float:left;margin-top:10px;width:22px;height:20px;background:url("../Components/Menu/img/cartIcon.png") no-repeat center}.cart__text{margin-top:12px;margin-left:8px;font-size:11px;font-weight:700;float:left}.role__text{text-transform:uppercase;margin-left:8px;font-size:11px;font-weight:700;margin-bottom:15px}.cart:hover{cursor:pointer}.cart__counter{margin-top:7px;margin-left:8px;font-size:11px;font-weight:700;padding:5px 7px;float:left;border-radius:50%;width:24px;height:24px;background-color:#e3e8f0}.signInButton{font-size:11px;letter-spacing:1px;font-weight:700;padding:7px 0px;width:85px;height:32px;border-radius:16px;background-color:#ffffff;border:2px solid #e3e8f0}.auth-button{font-size:11px;letter-spacing:1px;font-weight:700;padding:7px 0px;width:85px;height:32px;border-radius:16px;background-color:#ffffff;border:2px solid #e3e8f0;outline:0 !important;margin-top:-15px}.auth-button:hover{color:#b9bed1}.devider{height:1px;background-color:#f0f2f7}.footer-menu__item{margin-top:17px;float:left;text-transform:uppercase;font-size:11px;letter-spacing:1px;color:#b9bed1;font-weight:700;margin-right:30px}.footer-menu__item:hover{color:#343e5c}.footer-bar__icon{margin-top:15px;float:left;width:17px;height:17px}.footer-bar__input{margin-top:12px;margin-left:4px;outline:none;border:none}.footer-bar__input::placeholder{font-size:14px;color:#a1a8bd;font-weight:800}.filter__item{font:100% "Montserrat", sans-serif;text-transform:uppercase;font-size:11px;letter-spacing:1px;color:#b9bed1;font-weight:700}.ui.secondary.menu .item{font:100% "Montserrat", sans-serif;text-transform:uppercase;font-size:11px;letter-spacing:1px;color:#b9bed1;font-weight:700}.main-content__table{background-color:white;padding:50px 20px;margin-top:50px;height:480px}.main-content__table table{margin:0px auto}th{width:250px;border:1px solid #dfe4ee;background-color:#f0f2f7}td{width:250px;border:1px solid #dfe4ee}.footer{height:150px;background-color:#222328}.footer__logo{margin-top:60px;width:177px;height:37px;background:url("../Components/Footer/img/logoFooter.png") no-repeat center}.footer__menu{margin-top:60px;margin-left:92px}.item{text-transform:uppercase;margin-left:25px;margin-top:5px;float:left;font-size:12px;color:#ffffff;font-weight:700}.item:hover{color:#b9bed1}.social{margin-top:60px;margin-left:20px}.social__icon_facebook{width:24px;height:24px;background:url("../Components/Footer/img/FacebookIcon.png") no-repeat center;float:left}.social__icon_twitter{margin-left:33px;width:24px;height:24px;background:url("../Components/Footer/img/TwitterIcon.png") no-repeat center;float:left}.social__icon_google{margin-left:33px;width:26px;height:24px;background:url("../Components/Footer/img/G+Icon.png") no-repeat center;float:left}#head{background:url("../Components/Home/img/background.png") no-repeat center center;height:680px;padding-top:204px}.title__text_large{font-size:90px;letter-spacing:41px;line-height:82px;color:#fff;font-family:"Montserrat";text-align:center}.title__text_small{margin-top:15px;font-size:55px;letter-spacing:10px;line-height:82px;color:#fff;font-family:"Montserrat";text-align:center;text-transform:uppercase}.title__arrow{margin:0px auto;margin-top:150px;width:45px;height:45px;border-radius:23px;border:1px solid #ffffff;color:white;font-size:20px;padding:9px 4px}#advantages{height:642px;background-color:white}.advantages__title{font-size:42px;letter-spacing:4px;line-height:17px;color:black;text-align:center;margin-top:98px}.advantage{margin-top:71px;padding-top:57px;height:357px;box-shadow:1px 1.732px 4px rgba(149,147,147,0.33);background-color:#f4f4f4}.advantage__image_hand{height:78px;width:100px;margin:0px auto;background:url("../Components/Home/img/Hand.png") no-repeat center center}.advantage__image_property{height:78px;width:100px;margin:0px auto;background:url("../Components/Home/img/EditProperty.png") no-repeat center center}.advantage__image_conference{height:78px;width:100px;margin:0px auto;background:url("../Components/Home/img/Conference.png") no-repeat center center}.advantage__title{margin-top:28px;font-size:20px;letter-spacing:2px;line-height:17px;text-align:center}.advantage__title:after{content:'';display:block;width:37px;height:1px;background-color:#000000;opacity:20%;margin:0px auto;margin-top:20px}.advantage__text{margin:0px auto;margin-top:20px;width:292px;font-size:17px;line-height:22px;color:black;font-family:"Montserrat";text-align:center}#login{height:731px;background:url("../Components/Login/img/blackBackground.png") no-repeat center center}.form__title{font-size:42px;letter-spacing:4px;line-height:49px;color:#fff;font-family:"Montserrat";text-align:center;text-transform:uppercase;width:714px;margin:0px auto;margin-top:71px}.form__subtitle{font-size:20px;letter-spacing:3px;line-height:25px;color:#fff;font-family:"Montserrat";text-align:center;text-transform:uppercase;width:313px;margin:0px auto;margin-top:21px}.form__link{font-size:15px;letter-spacing:3px;line-height:25px;color:#fff;font-family:"Montserrat";text-align:center;text-transform:uppercase;width:313px;margin:0px auto;margin-top:21px}.form__link:hover{color:black}.form__arrow{padding-top:5px;padding-left:2px;font-size:15px;width:33px;height:33px;border-radius:16px;background-color:#black;border:1px solid #ffffff;border-style:dashed;margin:0px auto;margin-top:31px;color:white}.loginForm__input{margin:0px auto;margin-top:9px;margin-bottom:9px;width:370px;height:67px;border-radius:3px;background-color:black;border:1px solid #ffffff;padding-left:35px;font-size:13px;letter-spacing:1px;line-height:35px;color:white;font-family:"Montserrat";display:block}.loginForm__input::placeholder{font-size:13px;letter-spacing:1px;line-height:35px;color:white;font-family:"Montserrat";text-transform:uppercase;opacity:50%}.loginForm__input:focus{width:370px;height:67px;border-radius:3px;background-color:black;border:1px solid #ffffff;padding-left:35px;font-size:13px;letter-spacing:1px;line-height:35px;color:white;font-family:"Montserrat";box-shadow:0 0 35px rgba(14,14,14,0.3);margin:0px auto;display:block;margin-top:9px;margin-bottom:9px}.loginForm__input_act::placeholder{font-size:13px;letter-spacing:1px;line-height:35px;color:white;font-family:"Montserrat";text-transform:uppercase}.loginForm{margin-top:30px}.loginForm__btn{width:369px;height:70px;display:block;margin:0px auto;margin-top:20px;border-radius:4px;border:2px solid #ffffff;background-color:rgba(0,0,0,0);font-size:15px;letter-spacing:2px;line-height:35px;color:#fff;text-align:center;text-transform:uppercase}.loginForm__btn:hover{background-color:white;color:black}.regForm__btn{width:369px;height:70px;display:block;margin:0px auto;border-radius:4px;border:2px solid #ffffff;background-color:rgba(0,0,0,0);font-size:15px;letter-spacing:2px;line-height:35px;color:#fff;text-align:center;text-transform:uppercase}.regForm__btn:hover{background-color:white;color:black}.shop__addButton{font-size:11px;letter-spacing:1px;font-weight:700;padding:7px 0px;width:85px;height:32px;border-radius:16px;background-color:#ffffff;border:2px solid #e3e8f0}.about{height:800px;background-color:white;padding-top:1px}.about-header{color:#22241b;font-family:"Montserrat";font-size:37px;font-weight:400;line-height:55px;text-transform:uppercase;letter-spacing:4.44px;margin-top:76px}.left-block{float:left;border:1px red solid}.background{margin-top:42px;float:left}.about__title{opacity:0.66;color:#22241b;font-family:"Montserrat";font-size:22px;font-weight:400;line-height:55px;text-transform:uppercase;letter-spacing:2.64px}.background__content{width:432px;color:#22241b;font-family:"Montserrat";font-size:15px;font-weight:400;line-height:27px;letter-spacing:0.3px;margin-top:27px}.right-block{float:left;margin-left:48px;margin-top:42px}.rhombuses{margin-top:18px;margin-left:19px}.rhombuses__item{float:left;margin-right:54px}.rhombuses__item:nth-child(3){margin-right:0px}.rhombus{box-sizing:border-box;background:url(../Components/Home/img/rhombus.png) no-repeat center;width:105px;height:105px;color:#204d15;font-family:"Montserrat";font-size:20px;font-weight:400;line-height:55px;text-transform:uppercase;letter-spacing:2.4px;text-align:center;display:table-cell;vertical-align:middle}.signature{width:105px;height:27px;color:#22241b;font-family:"Montserrat";font-size:12px;font-weight:700;line-height:18px;text-transform:uppercase;letter-spacing:0.24px;text-align:center;margin-top:25px}.contact{margin-top:57px;margin-left:2px}.contact__items{margin-top:12px;float:left}.contact__item{color:#204d15;font-family:"Montserrat";font-size:14px;font-weight:400;letter-spacing:1.68px;margin-bottom:21px}.contact__item:nth-child(3){margin-bottom:0px}.contact__block{margin-top:22px;margin-left:43px;float:left}.contact__img{background:url(../Components/Home/img/icon.png) no-repeat center top;float:left;width:64px;height:64px}.contact__subscript{opacity:0.72;color:#22241b;font-family:"Montserrat";font-size:14px;font-weight:700;font-style:italic;line-height:55px;letter-spacing:0.28px;float:left;margin-top:3px;margin-left:20px}.product__rating{float:right}.Star{font-size:2em;color:#cfd8dc;cursor:pointer;transition:all 0.3s}.main-content__userData{width:500px;margin:0px auto}.main-content__title{text-align:center;color:#22241b;font-family:"Montserrat";font-size:24px;margin-bottom:25px}.main-content{height:550px;background-color:white;border-radius:15px;margin-top:25px;margin-bottom:25px;padding-top:25px}.ui.black.button,.ui.black.buttons .button{width:180px;text-shadow:none;background-image:none}.ui.input{margin-top:15px}.btn-link,.btn-link:hover{color:black}.btn-primary{color:black;background-color:white;border-color:black}.btn-primary:hover{color:white;background-color:black;border-color:black}.userPage__button{margin-top:25px;margin-bottom:25px}.userPage__modal{position:relative !important;width:80%;margin-top:400px;height:200px}.modals.dimmer .ui.scrolling.modal{margin:17rem auto}.table__button{padding-top:10px;padding-bottom:10px}.table__icon:hover{cursor:pointer}.product__buttons{float:right}.rating__icon{margin-top:5px}.pages{margin-top:30px;margin-bottom:30px}.pages__center-block{height:90px;border-radius:5px;background-color:#ffffff}.prev-icon{margin-top:39px;margin-left:50px;width:10px;height:14px;background:url("../Components/Shop/Cards/img/PrevIcon.png") no-repeat center;float:left;cursor:pointer}.page-numbers{margin:0px auto}.number-selected{width:48px;height:48px;border-radius:24px;filter:drop-shadow(0px 20px 20px rgba(168,172,185,0.62));background-color:#ffffff;z-index:2;opacity:70%}.next-icon{margin-top:39px;width:10px;height:14px;background:url("../Components/Shop/Cards/img/NextIcon.png") no-repeat center;float:left;cursor:pointer}.page-number{margin-top:20px;width:48px;height:48px;font-size:14px;color:#3a405b;font-weight:700;text-align:center;float:left;z-index:4;cursor:pointer;padding-top:15px}.main-content__goods{background-color:white;padding:60px}.page-button{padding:0;border:none;font:inherit;color:inherit;background-color:transparent}.modal__button{padding-bottom:20px}.no-goods{text-transform:uppercase;font-family:"Montserrat";font-size:24px;width:400px;height:400px;margin:0px auto;padding-top:175px;margin-top:50px}.form__message{width:400px;height:30px;margin-left:90px;font-size:15px;color:red}.regForm{margin-top:-50px}.cart__count{font-size:15px;margin-right:10px}.cart__box{border:1px solid black}.card__img{max-width:100%}.addButton__text{font-size:11px;letter-spacing:1px;font-weight:700;text-transform:uppercase}.product__mark{font-size:22px;margin-left:5px;cursor:pointer}.rating__modal{position:relative !important;width:80%;margin-top:400px;height:auto}.rating__table{margin:0px auto}.product__tag{font-size:20px;margin-right:15px;float:left}.productCard__tag{font-size:15px;margin-right:5px;float:left}.productCard__tags{margin-top:5px}.product__tags{margin-top:5px;width:470px}.button__addTag{margin-bottom:5px}.modal__icon{cursor:pointer}.menu__roleButton{margin-top:-15px;margin-left:10px}
