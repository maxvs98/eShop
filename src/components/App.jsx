/*import React, { Component } from 'react';
import { Container } from 'semantic-ui-react';
import axios from 'axios';
import ProductCard from '../containers/ProductCard';
import Filter from '../containers/Filter';
import Menu from '../containers/Menu';
import { Card } from 'semantic-ui-react';

class App extends Component {
  componentDidMount() {
    const { setProducts } = this.props;
    axios.get('/products.json').then(({ data }) => {
      setProducts(data);
    });
  }

  render() {
    const { products, isReady } = this.props;

    return (
      <div>
        <div class="header__content">
          <Menu />
          <Container>
            <Filter />
          </Container>
        </div>
        <Container>
          <Card.Group itemsPerRow={4}>
            {!isReady
              ? 'загрузка'
              : products.map((product, i) => (
                <ProductCard key={i} {...product} />
              ))}
          </Card.Group>
        </Container>
      </div>
    );
  }
}

export default App;*/

import React, { Component } from 'react';
import { Container } from 'semantic-ui-react';
import axios from 'axios';
import ProductCard from '../containers/ProductCard';
import Filter from '../containers/Filter';
import Menu from '../containers/Menu';
import { Card } from 'semantic-ui-react';
import Shop from '../containers/Shop';
import Users from '../containers/Users';
import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';

class App extends Component {


  render() {
    return (
      <Router>
        <Switch>
          <Route path="/" exact component={Home}/>
          <Route path="/users" component={Users}/>
          <Route path="/shop" component={Shop}/>
        </Switch>
      </Router>
    );
  }
}

const Home = () => (
  <div>
    <div class="header__content">
      <Menu />
    </div>
    Home page
  </div>
);

export default App;
