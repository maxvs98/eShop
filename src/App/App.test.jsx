import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from './App';

import createStore from '../storage/store';

const store = createStore();

test('First snapshot test', () => {
  const component = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
