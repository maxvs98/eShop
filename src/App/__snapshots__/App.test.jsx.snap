// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot test 1`] = `
<div>
  <div
    className="header__content"
  >
    <div>
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-md-1 text-center"
          >
            <a
              href="/"
            >
              <div
                className="menuIcon"
              />
            </a>
          </div>
          <div
            className="col-md-3 text-center"
          >
            <a
              href="/"
              onClick={[Function]}
            >
              <img
                alt="logo"
                src="logo.png"
              />
            </a>
          </div>
          <div
            className="col-md-5"
          >
            <div
              className="menu clearfix"
            >
              
              
              <div />
              
            </div>
          </div>
          <div
            className="col-md-2"
          >
            <div
              className="cart clearfix"
            >
              <div
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
              >
                <div
                  className="cart__icon"
                />
                <div
                  className="cart__text"
                >
                  CART
                </div>
                <div
                  className="cart__counter"
                >
                  0
                </div>
              </div>
            </div>
          </div>
          <div
            className="col-md-1 text-center"
          >
            <div
              className="role__text"
            />
            <p>
              <a
                href="/login"
                onClick={[Function]}
              >
                <button
                  className="auth-button"
                  type="button"
                >
                  Sign in
                </button>
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <section
    id="head"
  >
    <div
      className="title"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-md-12 col-lg-12"
          >
            <div
              className="title__text_small"
            >
              Simple
            </div>
            <div
              className="title__text_large"
            >
              eShop
            </div>
            <div
              className="title__arrow"
            >
              <a
                href="/"
              >
                <div>
                  <i
                    aria-hidden="true"
                    className="grey angle down large inverted icon"
                    onClick={[Function]}
                  />
                </div>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    id="advantages"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-lg-12"
        >
          <div
            className="advantages__title"
          >
            НАШИ ПРЕИМУЩЕСТВА
          </div>
        </div>
        <div
          className="col-xs-12 col-sm-6 col-md-4 col-lg-4"
        >
          <div
            className="advantage"
          >
            <div
              className="advantage__image_hand"
            />
            <div
              className="advantage__title"
            >
              НАДЕЖНОСТЬ
            </div>
            <div
              className="advantage__text"
            >
              Мы всегда придем к вам на помощь в решении самых неоднозначных вопросов
            </div>
          </div>
        </div>
        <div
          className="col-xs-12 col-sm-6 col-md-4 col-lg-4"
        >
          <div
            className="advantage"
          >
            <div
              className="advantage__image_property"
            />
            <div
              className="advantage__title"
            >
              ДОСТОВЕРНОСТЬ
            </div>
            <div
              className="advantage__text"
            >
              Достоверность - главный принцип нашей работы
            </div>
          </div>
        </div>
        <div
          className="col-xs-12 col-sm-12 col-md-4 col-lg-4"
        >
          <div
            className="advantage"
          >
            <div
              className="advantage__image_conference"
            />
            <div
              className="advantage__title"
            >
              КОМПЕТЕНТНОСТЬ
            </div>
            <div
              className="advantage__text"
            >
              Наши специалисты с большим опытом работы помогут разобраться во всех тонкостях
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    className="container-fluid"
  >
    <div
      className="devider"
    />
  </div>
  <section
    className="about clearfix"
  >
    <div
      className="container clearfix"
    >
      <div
        className="about-header"
      >
        о нас
      </div>
      <div
        className="background"
      >
        <div
          className="about__title"
        >
          бэкграунд
        </div>
        <div
          className="background__content"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis.
        </div>
        <div
          className="background__content"
        >
          Ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui.
        </div>
        <div
          className="background__content"
        >
          Ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa quiis aute irure dolor in reprehenderit in is aute irure dolor in reprehenderit in is aute irure dolor in reprehenderit in.
        </div>
      </div>
      <div
        className="right-block"
      >
        <div
          className="merit clearfix"
        >
          <div
            className="about__title"
          >
            Заслуги
          </div>
          <div
            className="rhombuses clearfix"
          >
            <div
              className="rhombuses__item"
            >
              <div
                className="rhombus"
              >
                75
              </div>
              <div
                className="signature"
              >
                Реализованных проектов
              </div>
            </div>
            <div
              className="rhombuses__item"
            >
              <div
                className="rhombus"
              >
                154
              </div>
              <div
                className="signature"
              >
                Положительных отзыва
              </div>
            </div>
            <div
              className="rhombuses__item"
            >
              <div
                className="rhombus"
              >
                8
              </div>
              <div
                className="signature"
              >
                Лет
                <br />
                опыта
              </div>
            </div>
          </div>
        </div>
        <div
          className="contact clearfix"
        >
          <div
            className="about__title"
          >
            Связь
          </div>
          <div
            className="contact__items"
          >
            <div
              className="contact__item"
            >
              skype.com/myname
            </div>
            <div
              className="contact__item"
            >
              vk.com/12343
            </div>
            <div
              className="contact__item"
            >
              info@sitename.ru
            </div>
          </div>
          <div
            className="contact__block"
          >
            <div
              className="contact__img"
            />
            <div
              className="contact__subscript"
            >
              Не стесняйтесь, пишите
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="footer"
    id="footer"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-3"
        >
          <div
            className="footer__logo"
          />
        </div>
        <div
          className="col-md-7"
        >
          <div
            className="footer__menu clearfix"
          >
            <a
              href="/"
            >
              <div
                className="item"
              >
                Home
              </div>
            </a>
            <a
              href="/shop"
            >
              <div
                className="item"
              >
                Shop
              </div>
            </a>
            <a
              href="/"
            >
              <div
                className="item"
              >
                Team
              </div>
            </a>
            <a
              href="/"
            >
              <div
                className="item"
              >
                About us
              </div>
            </a>
            <a
              href="/"
            >
              <div
                className="item"
              >
                Contacts
              </div>
            </a>
          </div>
        </div>
        <div
          className="col-md-2"
        >
          <div
            className="social clearfix"
          >
            <a
              href="/"
            >
              <div
                className="social__icon_facebook"
              />
            </a>
            <a
              href="/"
            >
              <div
                className="social__icon_twitter"
              />
            </a>
            <a
              href="/"
            >
              <div
                className="social__icon_google"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
